lib/controllers/root-controller.js

```diff
@@ -488,8 +497,30 @@ export default class RootController extends React.Component {
 
   async viewChangesForCurrentFile(stagingStatus) {
     const editor = this.props.workspace.getActiveTextEditor();
-    const absFilePath = editor.getPath();
+    const absFilePath = await realPath(editor.getPath());
     const repoPath = this.props.repository.getWorkingDirectoryPath();
+    if (repoPath === null) {
+      const [projectPath] = this.props.project.relativizePath(editor.getPath());
+      const notification = this.props.notificationManager.addInfo(
+        "Hmm, we don't have anything to compare this to.",
```

Comment:
I :heart: the approachability of this message. The use of "we" reminds me of something @simurai brought up last week. IIRC, he asked whether Atom should ever say "I" or "we?" In other words, can Atom speak in the first person? To me, speaking in the first person gives the impression that Atom is a bot (using "I" or "me") or that Atom is a company or OSS team (using "we"). In my loosely-held opinion, I lean toward having Atom _not_ speak in the first person.

With that in mind, I'd propose something like:

> Hmm, there's nothing to compare this file to